rpm/argparse-manpage.spec.tpl

```diff
@@ -73,18 +74,25 @@ Requires: python2-%name = %version-%release
 
 %package -n     python2-%name
 Summary:        %{sum Python 2}
+Requires:       python2-setuptools
 
 %description -n python2-%name
 %{desc}
 
 
 %package -n     python3-%name
 Summary:        %{sum Python 3}
+Requires:       python3-setuptools
```

Comment:
Doesn't this completely override the entire point of this PR?

Reply:
perhaps on EL7 and EL8, what is the typical way around this, without pyproject macros?

Reply:
Ah, I mistakenly affected also the '%{with pyproject}' case.